packages/iTwinUI-react/src/core/TimePicker/TimePicker.tsx

```diff
@@ -383,6 +570,8 @@ const TimePickerColumn = <T,>(props: TimePickerColumnProps<T>): JSX.Element => {
     isSameSelected,
     setFocus = false,
     valueRenderer,
+    precision = 'minutes',
+    useCombinedRenderer = false,
     className = 'iui-time',
   } = props;
   const needFocus = React.useRef(setFocus);
```

Comment:
I couldn't highlight L581-585, but does that `useEffect` need a dependency array? Without that array, is the benefit of `useEffect` gone?

Reply:
So I think what you're thinking is that because there's no dependency array, that code is being run for every render, but the text says something about not doing something on every rerender, right? But I don't think it's about not running the code every rerender, but about not focusing on every rerender. Also I don't like messing with things that are already there, especially if I don't fully understand them.